server/chat-plugins/chatlog.ts

```diff
@@ -554,13 +550,13 @@ export const pages: PageTable = {
 
 		void accessLog.writeLine(`${user.id}: <${roomid}> ${date}`);
 		this.title = '[Logs] ' + roomid;
-		let cap = 'all';
+		let limit = null;
 		let search;
-		const [, term] = opts?.includes('search-') ? Chat.splitFirst(opts, 'search-') : [];
-		if (term) {
-			const [input, num] = term.split('&');
+		if (opts?.startsWith('search-')) {
+			const [input, limitString] = opts.split('--limit-');
```

Comment:
Oh whoops, I forgot to add special detection for `--all`, can you add that?

Reply:
Gotcha.